components/MaskedTransition/README.md

```diff
@@ -0,0 +1,130 @@
+<!--docs:
+title: "Masked Transitions"
+layout: detail
+section: components
+excerpt: "A masked transition reveals content from a source view using a view controller transition."
+iconId: maskedTransition
+path: /catalog/masked-transitions/
+api_doc_root: true
+-->
+
+# Masked Transitions
+
+A masked transition reveals content from a source view using a view controller transition.
+
+## Design & API Documentation
+
+<ul class="icon-list">
+  <li class="icon-list-item icon-list-item--spec"><a href="https://material.io/guidelines/motion/transforming-material.html#transforming-material-radial-transformation">Material Design guidelines: Radial Transformations</a></li>
+  <li class="icon-list-item icon-list-item--link"><a href="https://material.io/components/ios/catalog/masked-transitions/api-docs/Classes/MDCMaskedTransition.html">API: MDCMaskedTransition</a></li>
+</ul>
+
+- - -
+
+## Installation
+
+### Requirements
+
+- Xcode 7.0 or higher.
+- iOS SDK version 7.0 or higher.
+
+### Installation with CocoaPods
+
+To add this component to your Xcode project using CocoaPods, add the following to your `Podfile`:
+
+```
+pod 'MaterialComponents/MaskedTransition'
+```
+<!--{: .code-renderer.code-renderer--install }-->
+
+Then, run the following command:
+
+``` bash
+pod install
+```
+
+- - -
+
+## Overview
+
+A masked transition is a UIViewController transition using the
+[Transitioning](https://github.com/material-motion/transitioning-objc) library. It assumes that you
```

Comment:
`Transitioning` is more or less an implementation detail at this point. Please explain why someone would want to use this from a client's perspective.